web/src/popover_menus.js

```diff
@@ -91,6 +91,11 @@ export function popover_items_handle_keyboard(key, $items) {
 
     if (index === -1) {
         index = 0;
+    }
+
+    if (key === "enter") {
+        $items.eq(index).trigger("click");
+        return;
```

Comment:
This looks like a bug I introduced in 8eba95e556bad9e2294a15ae21fadeafdf075348. 

Looks like the correct code is `if (key === "enter" && index >= 0 && index < $items.length) {`